src/dataflow/analysis/graph-internals.ts

```diff
@@ -18,9 +18,121 @@
  * of Node/Edge like ParticleNode, etc.
  */
 
-import {Claim} from '../../runtime/particle-claim.js';
+import {Claim, ClaimType} from '../../runtime/particle-claim.js';
 import {Check} from '../../runtime/particle-check.js';
 
+/** Represents the accumulation of tags/nodes/edges that flow along a path in the graph. */
```

Comment:
Nodes and edges don't flow. Reword this comment. I suggest: "Represents a path in the graph and the set of tags and checks that flow along it."

Reply:
Done